tests/queries/0_stateless/02563_arrow_low_cardinality_bug.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
```

Comment:
Why no-fasttest?

Reply:
Build for fasttest doesn't contain arrow library and Arrow format